### PR TITLE
Fixes for flake8

### PR DESCRIPTION
**Pull request by dave3d**

### CODE REVIEW

File: utils/myspell.py [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
 
     lc = 1
     for x in fp:
-      spell_checker.set_text(x)
-      for error in spell_checker:
-        print("Error:", error.word, "(line", lc, ")")
-      lc = lc+1
+        spell_checker.set_text(x)
+        for error in spell_checker:
+            print("Error:", error.word, "(line", lc, ")")
```
Comment (zivy) [inline utils/myspell.py:20]: Should this print to stderr?
print("Error:", error.word, "(line", lc, ")", file=sys.stderr)

File: utils/myspell.py [hunk 2]
```diff
@@ -1,27 +1,25 @@
...
+        lc = lc+1
 
 
-#myspell('/Users/dchen/SimpleITK/Documentation/docs/source/faq.rst')
-myspell('/Users/dchen/SimpleITK/Code/Common/include/sitkImage.h')
-
+if __name__ == '__main__':
+    myspell('/Users/dchen/SimpleITK/Code/Common/include/sitkImage.h')
```
Comment (zivy) [inline utils/myspell.py:25]: Get image from argv, don't hard code it.

File: utils/myspell.py (hunk 1 above)
Comment (dave3d) [reply utils/myspell.py]: done

File: utils/myspell.py (hunk 2 above)
Comment (dave3d) [reply utils/myspell.py]: done

**Review by zivy — APPROVED** [review]

**Pull request merged**